@types/@elastic/apm-rum/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '@elastic/apm-rum' {
+  export interface UserObject {
+    id?: string | number
+    username?: string
```

Comment:
I believe the `username` is no longer needed with the latest approach. `Email` is needed for the `domain`, but `username` is not used anywhere.